tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc

```diff
@@ -290,6 +291,20 @@ bool EnergyPlusFixture::has_dfs_output(bool reset_stream)
     return has_output;
 }
 
+bool EnergyPlusFixture::match_err_stream(std::string const &expected_match, bool use_regex, bool reset_stream)
```

Comment:
@mbadams5 @nmerket let me know what you think about this function. If we like it, we can make similar functions for other streams.